src/main/java/core/basesyntax/dataservices/DataReaderImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax.dataservices;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataReaderImpl implements DataReader {
```

Comment:
The class `DataReaderImpl` should not begin with an empty line. Please remove the empty line to follow the checklist guidelines.